packages/components/psammead-section-label/src/titles.jsx

```diff
@@ -23,7 +23,7 @@ const paddingReverseDir = ({ dir }) =>
 // This makes it work right. I don't fully understand how, but am
 // eternally grateful to the Flexbugs project.
 // https://github.com/philipwalton/flexbugs#flexbug-3
-const FlexColumn = styled.div`
+const FlexColumn = styled.span`
```

Comment:
Can you check if we really need this `FlexColumn` or if we could try to merge it along with `FlexRow`, so we avoid having that many nested `span`s?

Reply:
If we remove `FlexColumn` or merge it with `FlexRow` everything still looks great on Chrome, FireFox and Safari. However, it causes the bar to get shifted down on Internet Explorer. And even though IE is a low priority browser for us, section labels are on every front page, so I don't think it's worth making this change just to avoid having a lot of `span`s in the DOM.

<img width="760" alt="Screenshot 2020-04-03 at 11 42 43" src="https://user-images.githubusercontent.com/17802955/78352699-b2034f80-75a0-11ea-9132-8a6774d124d5.png">

Reply:
All right, thanks for investing though :)